shell/platform/android/platform_view_android.cc

```diff
@@ -51,16 +52,10 @@ std::unique_ptr<AndroidSurface> AndroidSurfaceFactoryImpl::CreateSurface() {
       }
     case AndroidRenderingAPI::kVulkan:
       FML_DCHECK(enable_impeller_);
-      // TODO(kaushikiska@): Enable this after wiring a preference for Vulkan
-      // backend.
-#if false
-    return std::make_unique<AndroidSurfaceVulkanImpeller>(
+      FML_DLOG(ERROR) << "using vulkan";
```

Comment:
Remove this and add a "Using the Impeller Vulkan rendering backend" log to `android_surface_vulkan_impeller.cc` similar to the one for OpenGL ES